<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DnsWidgetClass</class>
 <widget class="QWidget" name="DnsWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DnsWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QLabel" name="label">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>DNS Server</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="server_edit"/>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="load_default_button">
       <property name="text">
        <string>Load Default</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_2">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>Request</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="request_edit"/>
     </item>
     <item row="1" column="3">
      <widget class="QCheckBox" name="reverse_checkbox">
       <property name="text">
        <string>Reverse Lookup</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="3">
      <layout class="QHBoxLayout" name="advanced_layout"/>
     </item>
     <item row="2" column="1" colspan="3">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
